model/src/cfd.rs

```diff
@@ -2376,6 +2405,23 @@ impl Dlc {
 
         Ok(Ok(Ok(signed_cet)))
     }
+
+    /// All the oracle event IDs associated with the DLC.
+    ///
+    /// This includes:
+    /// - the settlement event; and
+    /// - all the liquidation events that precede it.
+    pub fn event_ids(&self) -> Vec<BitMexPriceEventId> {
+        self.cets.keys().copied().collect_vec()
+    }
+
+    pub fn liquidation_event_ids(&self) -> Vec<BitMexPriceEventId> {
```

Comment:
Took me a while, is my understanding correct that a `liquidation_event_id` is every id which is not the `settlement_id`, i.e. the last one? 
It might be worth adding a comment.

Reply:
Correct. I'll add a comment!